.github/workflows/builds_and_tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.60.0
+          toolchain: 1.67.0
```

Comment:
Should we change toolchain to 1.69.0?

Reply:
https://github.com/Finschia/wasmvm/blob/17789e046ea44d2807ba0f176bf48c2ad0473947/docs/COMPILER_VERSIONS.md#versions-in-use

Since the version is specified, I think it's fine to keep it the same as upstream.